index.html

```diff
@@ -1,117 +1,84 @@
-<!--Content and Technical Requirements
-
-Title - make it meaningful.
-Header element <h1> - This is the main headline; include your name and what this page is.
-Photo
-Article element containing your bio
-Article element with links to resources or sites you like
-Article element with two sections for future projects (placeholders)
-Footer element with email and professional social media links; Github, twitter, LinkedIn, etc.
-Semantic mark-up for all major elements.
-1. Validate your html page with W3 validator: https://validator.w3.org/
-Appropriate folder structure: images, css.
-Notes -->
-
-
-
-
 <!DOCTYPE html>
 <html lang="en">
 <head>
 	<title> $ ~/Mitchell  </title>
-	<!--rotating background video  <video> </video> --> 
 	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.min.css">
 	<link rel="stylesheet" type="text/css" href="css/quiz.css">
-
 </head>
 
-<header> <h1> Shawn | Mitchell </h1> 
-			
-
+<header> 
+<h1> Shawn | Mitchell </h1> 
 </header>
   	
  	
 <body>
-<main> 
-	
-	<!---<nav> 
-		<ul>	
-			<li href="#main"> Main </li>
-
-			<li href="#projects"> Projects </li>
-			<li href="#contact"> Contact </li>
-		</ul>
-	</nav> -->
-	
+	<main> 
```

Comment:
If everything in the page is a child of this `main` div, then everything needs to be nested in it by indenting them. You are doing that up to line 19, but then your `article` tag pops out to the same indentation level as `main`